### ISSUE TITLE
Fabric 1.14.4 crash

### ISSUE THREAD

**Issue by Kermalis** [maruohon/litematica#65]
Hello, I get this crash a few minutes after loading a large schematic. I tried raising the amount of ram Minecraft has, but that didn't seem to solve the issue.

```
---- Minecraft Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 8/26/19 2:54 PM
Description: Batching chunks

java.lang.UnsupportedOperationException
	at fi.dy.masa.litematica.render.schematic.ChunkRenderDataSchematic$1.setOverlayTypeStarted(ChunkRenderDataSchematic.java:34)
	at fi.dy.masa.litematica.render.schematic.ChunkRendererSchematicVbo.renderOverlay(ChunkRendererSchematicVbo.java:436)
	at fi.dy.masa.litematica.render.schematic.ChunkRendererSchematicVbo.renderBlocksAndOverlay(ChunkRendererSchematicVbo.java:423)
	at fi.dy.masa.litematica.render.schematic.ChunkRendererSchematicVbo.rebuildChunk(ChunkRendererSchematicVbo.java:298)
	at fi.dy.masa.litematica.render.schematic.ChunkRenderWorkerLitematica.processTask(ChunkRenderWorkerLitematica.java:103)
	at fi.dy.masa.litematica.render.schematic.ChunkRenderWorkerLitematica.run(ChunkRenderWorkerLitematica.java:47)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at fi.dy.masa.litematica.render.schematic.ChunkRenderDataSchematic$1.setOverlayTypeStarted(ChunkRenderDataSchematic.java:34)
	at fi.dy.masa.litematica.render.schematic.ChunkRendererSchematicVbo.renderOverlay(ChunkRendererSchematicVbo.java:436)
	at fi.dy.masa.litematica.render.schematic.ChunkRendererSchematicVbo.renderBlocksAndOverlay(ChunkRendererSchematicVbo.java:423)

-- Affected level --
Details:
	All players: 2 total; [class_746['Kermalis'/7515, l='MpServer', x=3055.92, y=195.51, z=112.20], class_745['tukku473'/184, l='MpServer', x=3065.41, y=66.00, z=230.66]]
	Chunk stats: Client Chunk Cache: 529, 323
	Level dimension: minecraft:overworld
	Level name: MpServer
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: {}
	Level spawn location: World: (-16,64,48), Chunk: (at 0,4,0 in -1,3; contains blocks -16,0,48 to -1,255,63), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 22458110 game time, 26781213 day time
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: spectator (ID 3). Hardcore: false. Cheats: false
	Server brand: vanilla
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:574)
	at net.minecraft.class_310.method_1587(class_310.java:1923)
	at fi.dy.masa.litematica.render.schematic.ChunkRenderWorkerLitematica.run(ChunkRenderWorkerLitematica.java:57)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1176703240 bytes (1122 MB) / 4294967296 bytes (4096 MB) up to 4294967296 bytes (4096 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: fabric-loader-0.4.8+build.157-1.14.4
	LWJGL: 3.2.2 build 10
	OpenGL: NO CONTEXT
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Resource Packs: vanilla, file/JUM.zip, file/BrothersJohnsonUnshaders.zip (incompatible)
	Current Language: English (US)
	CPU: 8x Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz```

**Comment by vacla** [comment]
have you tried pasting it in creative mode?

**Comment by maruohon** [comment]
I haven't seen that crash before. Can you send/link the schematic you used to get this crash? Which other mods do you have? (Optifine?)

**Comment by vacla** [comment]
have you tried without optifabric?

**Comment by SunSerega** [comment]
Optifine doesn't work without optifabric, so i also have it and haven't had this problem. So problem is with schematics...

**Comment by SunSerega** [comment]
Oh, i think i just got the same crash, with half as big of a schematic...

[crash-2019-08-28_15.16.17-client.txt](https://github.com/maruohon/litematica/files/3550815/crash-2019-08-28_15.16.17-client.txt)

[latest.log](https://github.com/maruohon/litematica/files/3550816/latest.log)

[temp.zip](https://github.com/maruohon/litematica/files/3550819/temp.zip)

**Comment by maruohon** [comment]
This is starting to look like a possible threading issue, as it's somehow trying to use the static `EMPTY` chunk render data object for rendering... :thinking: Although I'll need to take a look at it in the IDE when I get my dev PC back set up.

**Comment by maruohon** [comment]
I think this crash should be fixed in the latest December build for 1.14.4+. The 1.12.x and 1.13.2 versions still need the fix implemented.
